BNA/src/Graphics/Effect.bf

```diff
@@ -402,7 +402,7 @@ namespace BNA.Graphics
 			let param = GetParameter(paramName);
 			CheckParam!(param, ShaderParameterClass.Vector, ShaderParameterType.float);
 
-			if(param.Elements != 3) return .Err;
+			//if(param.Elements != 3) return .Err;
```

Comment:
Should this be commented, or would it be better to replace with something like `if(param.Elements < 3)`?

Reply:
I just dug into this and looks like indeed MOJOShader is reporting the number of elements as "columns" rather than "elements". So go ahead and leave as-is, I'll fix all of these later.